theseus_gui/src/pages/Settings.vue

```diff
@@ -1,7 +1,471 @@
+            <Button
+              :disabled="settings.java_globals.JAVA_8.path === java8Install.path"
+              @click="() => setJava8Install(java8Install)"
+            >
+              <span v-if="settings.java_globals.JAVA_8.path === java8Install.path"
+                ><CheckIcon />Selected</span
+              >
+              <span v-else><PlusIcon />Select</span>
+            </Button>
+          </div>
+        </div>
+      </div>
+    </Modal>
+    <Card class="theming">
+      <h2>Themes</h2>
+      <div class="toggle-setting" style="display: flex">
+        <div class="description">
+          <h3>Color theme</h3>
+          <p>Change the global launcher color theme.</p>
+        </div>
+        <DropdownSelect
+          name="Theme dropdown"
+          :options="themeStore.themeOptions"
+          :default-value="settings.theme"
+          :model-value="settings.theme"
+          class="theme-dropdown"
+          @change="handleTheme"
+        />
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Java</h2>
+      <div class="settings-group">
+        <h3>Java 17 Location</h3>
+        <div class="toggle-setting">
+          <input
+            v-model="settings.java_globals.JAVA_17.path"
+            type="text"
+            class="input installation-input"
+            placeholder="/path/to/java17"
+          />
+          <span class="installation-buttons">
+            <Button @click="() => detectJava17Modal.show()">
+              <SearchIcon />
+              Auto Detect
+            </Button>
+            <Button @click="handleJava17FileInput">
+              <BrowseIcon />
+              Browse
+            </Button>
+            <Button @click="saveJavaPath">
+              <PlayIcon />
+              Test
+            </Button>
+          </span>
+        </div>
+      </div>
+      <div class="settings-group">
+        <h3>Java 8 Location</h3>
+        <div class="toggle-setting">
+          <input
+            v-model="settings.java_globals.JAVA_8.path"
+            type="text"
+            class="input installation-input"
+            placeholder="/path/to/java8"
+          />
+          <span class="installation-buttons">
+            <Button @click="() => detectJava8Modal.show()">
+              <SearchIcon />
+              Auto Detect
+            </Button>
+            <Button @click="handleJava8FileInput">
+              <BrowseIcon />
+              Browse
+            </Button>
+            <Button @click="saveJavaPath">
+              <PlayIcon />
+              Test
+            </Button>
+          </span>
+        </div>
+      </div>
+      <hr class="card-divider" />
+      <div class="settings-group">
+        <h3>Java Arguments</h3>
+        <input
+          v-model="settings.custom_java_args"
+          type="text"
+          class="input installation-input"
+          placeholder="-Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled"
```

Comment:
The placeholders should be more like "Enter java arguments...", it's more consistent with how we do them elsewhere